owncloudApp/src/main/java/com/owncloud/android/ui/activity/UploadPathActivity.java

```diff
@@ -57,6 +57,7 @@ protected void onCreate(Bundle savedInstanceState) {
     protected void onAccountSet(boolean stateWasRecovered) {
         super.onAccountSet(stateWasRecovered);
         if (getAccount() != null) {
+            // TODO: Check how to solve glitch when accessing the folder picker from camera uploads, sending the app to background, returning and avoid falling back to initial folder
```

Comment:
We can remove this todo and add it as known issue in the issue or PR to keep track of it. 👍

Reply:
Yeah I simply removed it, I was discussing about it with our great QA team (@jesmrec 😉) and it is an acceptable behaviour